contribs/gmf/src/disclaimer/component.js

```diff
@@ -415,10 +408,7 @@ DisclaimerController.prototype.closeDisclaimerMessage_ = function (layerUid, msg
  * Example:
  *
  *      <gmf-disclaimer
- *        gmf-disclaimer-map="::ctrl.map"
- *        gmf-disclaimer-external="::true"
- *        gmf-disclaimer-external-msg="disclaimerMsg"
- *        gmf-disclaimer-external-visibility="disclaimerVisibility">
```

Comment:
` gmf-disclaimer-external-visibility="disclaimerVisibility"` still exists, is optional but description of this example (just above) say that we configure it (as far as i undestand this desctiption)

Reply:
+1 Thanks :-)